src/index.html

```diff
@@ -1,13 +1,61 @@
+<!--#region head -->
 <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;500;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
+  <!--#endregion -->
+
   <body>
-    <h1>Product cards</h1>
+    <!--#region card head -->
+    <div class="product-card" data-qa="card">
+      <img
+      src="./images/imac.jpeg"
+      alt="apple A1419 imac-27 Retina 5K monoblock"
+      class="product-card__image"
+      >
```

Comment:
```suggestion
      <img
        src="./images/imac.jpeg"
        alt="apple A1419 imac-27 Retina 5K monoblock"
        class="product-card__image"
      >
```
Checklist:
14. [CODE STYLE] - If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.